tasks/internal/create-advisory-task/create-advisory-task.yaml

```diff
@@ -221,15 +227,30 @@ spec:
               echo "Remaining entries after filtering:"
               cat "$CONTENT_FILE"
 
+              CONTENT_BEFORE_COUNT=$(echo "$CONTENT_BEFORE_FILTER" | jq 'length')
```

Comment:
`jq 'length' <<< "$CONTENT_BEFORE_FILTER"` is what shellcheck has recommended in a bunch of other spots